File: app/src/main/java/com/zhai/mvpdemo/view/MainView.java
package com.zhai.mvpdemo.view;

import java.util.List;

/**
 * Created by zhai on 16/6/18.
 */

public interface MainView {

    void initRecylerView();

    void setContentView();

    void updateRecylerView(List<String> noteList);

    void clearNoteText();

    void initButton();
}

File: app/src/main/java/com/zhai/mvpdemo/view/MainView.kt
package com.zhai.mvpdemo.view

/**
 * Created by zhai on 16/6/18.
 */

interface MainView {

    fun initRecylerView()

    fun setContentView()

    fun updateRecylerView(noteList: List<String>)

    fun clearNoteText()

    fun initButton()
}
